.github/workflows/nccl-tests.yml

```diff
@@ -0,0 +1,41 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+    - msccl/*
+
+jobs:
+  nccl-tests:
+    name: Build and run nccl tests
+    runs-on: [self-hosted, linux, x64, gpu]
+    container:
+      image: nvcr.io/nvidia/pytorch:19.12-py3
+      options: --privileged --ipc=host --gpus=all
+    steps:
+    - name: Checkout msccl
+      uses: actions/checkout@v2
+    - name: Checkout nccl-tests
+      uses: actions/checkout@v2
+      with:
+        repository: nvidia/nccl-tests
```

Comment:
is it possible to also fork nccl-tests? for reasons I have to explain in a meeting.